services-js/commissions-app/src/client/common/CommentInput.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+export interface Props {
+  name: string;
+  placeholder: string;
+  title?: string;
+  error?: string;
+  value: string;
+  onChange: any;
+  onBlur: any;
+}
+
+export default class Comments extends React.Component<Props> {
+  render() {
+    return (
+      <div className="txt">
+        <textarea
+          rows={10}
+          title={this.props.title ? this.props.title : undefined}
```

Comment:
Where did the `<label>` go? Take a look at `TextInput` to see how it used `this.props.title` to put in the label field.

Reply:
No worries. Can you get it back in?